lib/postgrex/types.ex

```diff
@@ -292,10 +292,12 @@ defmodule Postgrex.Types do
 
   defp fetch!(table, oid) do
     case :ets.lookup(table, oid) do
+      [{_, info, nil}] ->
+        raise ArgumentError, "no extension found for oid `#{oid}`: " <> inspect(info)
       [value] ->
         value
       [] ->
-        raise ArgumentError, message: "no extension found for oid `#{oid}`"
+        raise ArgumentError, "no extension found for oid `#{oid}`"
```

Comment:
I have a test that I didn't commit that prints some information for "postgrex_test", it has 386 oids and uses 22340 memory. master has 224 oids and uses 12419 memory. This branch has 339 oids and uses 19995 memory. Memory is in words and I'm on 64bit and using 9.4.5